common/nymsphinx/src/utils/mod.rs

```diff
@@ -1,2 +1,33 @@
-pub mod encapsulation;
-pub mod poisson;
+// Copyright 2020 Nym Technologies SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use rand::Rng;
+use rand_distr::{Distribution, Exp};
+use std::time;
+
+// TODO: ask @AP why we are actually using Distribution::Exp(1/L) rather than just
```

Comment:
@aniampio I'm almost certain I've already asked this when we were initially working on Sphinx, but could you explain it again?